sktime/datatypes/_utilities.py

```diff
@@ -45,10 +49,17 @@ def get_time_index(X):
             return X.index
     # numpy3D and np.ndarray
     elif isinstance(X, np.ndarray):
-        return _get_index(X)
+        # np.ndarray
+        if X.ndim < 3:
```

Comment:
That's a repetition of `_get_index` function. I think you can leave
```
elif isinstance(X, np.ndarray):
    return _get_index(X)
```
as it was before.

Reply:
well, I thought, it was an unnecessary duplication of `get_time_index`, so I removed it in order to have the logic in one place.
I left `_get_index` in existence (and fixed it) since it was being called from elsewhere.